bVNC/src/main/java/com/iiordanov/bVNC/RemoteCanvas.java

```diff
@@ -245,10 +248,12 @@ public void run() {
         pd.setCanceledOnTouchOutside(false);
     }
 
-    RemotePointer init(final Connection settings, final Handler handler, final Runnable setModes) {
+    void init(final Connection settings, final Handler handler, final Runnable setModes, final Runnable clearToggles, final String vvFileName) {
```

Comment:
We don't want to handle reauth for vvFile because it simply means the vv file expired.